$DragonFly: src/gnu/usr.bin/cc34/cc_prep/patches/Attic/config,elfos.h.patch,v 1.1 2004/06/20 21:41:15 joerg Exp $

--- config/elfos.h.orig	2004-06-20 23:38:49.000000000 +0200
+++ config/elfos.h	2004-06-20 23:39:04.000000000 +0200
@@ -75,7 +75,7 @@
    default to dwarf2.  */
 
 #ifndef PREFERRED_DEBUGGING_TYPE
-#define PREFERRED_DEBUGGING_TYPE DWARF2_DEBUG
+#define PREFERRED_DEBUGGING_TYPE DBX_DEBUG
 #endif
 
 /* All SVR4 targets use the ELF object file format.  */
